src/components/MessageList.vue

```diff
@@ -423,6 +425,38 @@ export default {
         scrollToBottom() {
             this.$el.scrollTop = this.$el.scrollHeight;
         },
+        scrollToUnread() {
+            let messageId = this.buffer.last_read ? -1 : this.filteredMessages[0].id;
+
+            // Find the message id for unread marker
+            for (let i = 0; i < this.filteredMessages.length; i++) {
+                if (this.shouldShowUnreadMarker(i)) {
+                    let idx = (i - 1 >= 0) ? i - 1 : 0;
```

Comment:
Could do with a quick comment on what this calculation is for

Reply:
wish i could remember what its for :(